orientdb/README.md

```diff
@@ -64,7 +69,29 @@ WARNING: Creating a new database will be done safely with multiple threads on a
 * ```orientdb.remote.storagetype``` - Storage type of the database on remote server
     * This is only required if using a ```remote:``` connection url
 
-## Known Issues
 
+## Plot results
+
+To plot workload results install gnuplot, then create a directory to collect the outputs
+```
+mkdir -p orientdb/outputs/orientdb
+```
+load the data
+```
+./bin/ycsb load orientdb -s -P workloads/workloadc 
+```
+and run the workload, e.g. `workloadc`, by following the naming convention for the file name `outputRun_<workload>.txt`:
+```
+./bin/ycsb run orientdb -s -P workloads/workloadc > orientdb/outputs/orientdb/outputRun_workloadc.txt
+```
```

Comment:
this looks like it's parsing out the default text measurements. it would be simpler to use the `exporter` settings to write e.g. a json file via `JSONArrayMeasurementsExporter` or `JSONMeasurementsExporter`